metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/topology/ParserTopologyBuilder.java

```diff
@@ -106,19 +105,22 @@ public static TopologyBuilder build(String zookeeperUrl,
   /**
    * Create a spout that consumes tuples from a Kafka topic.
    *
-   * @param zookeeperUrl            Zookeeper URL
+   * @param zkQuorum Zookeeper URL
    * @param sensorType              Type of sensor
-   * @param offset                  Kafka topic offset where the topology will start; BEGINNING, END, WHERE_I_LEFT_OFF
-   * @param kafkaSpoutConfigOptions Configuration options for the kafka spout
+   * @param kafkaConfigOptional     Configuration options for the kafka spout
    * @param parserConfig            Configuration for the parser
    * @return
    */
-  private static KafkaSpout createKafkaSpout(String zookeeperUrl, String sensorType, SpoutConfig.Offset offset, EnumMap<SpoutConfigOptions, Object> kafkaSpoutConfigOptions, SensorParserConfig parserConfig) {
-
+  private static StormKafkaSpout createKafkaSpout(String zkQuorum, String sensorType, Optional<Map<String, Object>> kafkaConfigOptional, SensorParserConfig parserConfig) {
```

Comment:
StormKafkaSpout's return type here will actually be StormKafkaSpout<Object, Object>, right?  Can we make that explicit, rather than untyped (and also drop the Object.class from the creates assuming the other typing change)?